Shared/Samples/Identify graphics/IdentifyGraphicsView.swift

```diff
@@ -0,0 +1,112 @@
+    @State private var map = Map(basemapStyle: .arcGISTopographic)
+    
+    /// A graphics overlay containing a yellow polygon graphic.
+    @State private var graphicsOverlay = {
+        let graphicsOverlay = GraphicsOverlay()
+        
+        // Create a polygon from a list of points.
+        let polygon = Polygon(points: [
+            Point(x: -20e5, y: 20e5),
+            Point(x: 20e5, y: 20e5),
+            Point(x: 20e5, y: -20e5),
+            Point(x: -20e5, y: -20e5)
+        ])
+        
+        // Create a graphic of the polygon and add it to the overlay.
+        let polygonGraphic = Graphic(geometry: polygon)
+        graphicsOverlay.addGraphic(polygonGraphic)
+        
+        // Create a renderer using a simple fill symbol.
+        let polygonSymbol = SimpleFillSymbol(style: .solid, color: .yellow)
+        graphicsOverlay.renderer = SimpleRenderer(symbol: polygonSymbol)
+        
+        return graphicsOverlay
+    }()
+    
+    /// The tap location's screen point used in the identify operation.
+    @State private var tapScreenPoint: CGPoint?
+    
+    /// A Boolean value indicating whether to show identify result alert.
+    @State private var isShowingIdentifyResultAlert = false
+    
+    /// The message shown in the identify result alert.
+    @State private var identifyResultMessage = "" {
+        didSet { isShowingIdentifyResultAlert = identifyResultMessage.isNotEmpty }
+    }
+    
+    /// A Boolean value indicating whether to show an error alert.
+    @State private var isShowingErrorAlert = false
+    
+    /// The error shown in the error alert.
+    @State private var error: Error? {
+        didSet { isShowingErrorAlert = error != nil }
+    }
+    
+    var body: some View {
+        MapViewReader { mapViewProxy in
+            MapView(map: map, graphicsOverlays: [graphicsOverlay])
+                .onSingleTapGesture { screenPoint, _ in
+                    tapScreenPoint = screenPoint
+                }
+                .task(id: tapScreenPoint) {
+                    guard let tapScreenPoint else { return }
+                    
+                    do {
+                        // Identify the screen point on the graphics overlay using the proxy.
+                        // Use `identifyGraphicsOverlays` instead if you need to identify on
+                        // all the graphics overlay present in the map view.
+                        let identifyResult = try await mapViewProxy.identify(
+                            on: graphicsOverlay,
+                            screenPoint: tapScreenPoint,
+                            tolerance: 12
+                        )
+                        
+                        // Display an alert if any graphics were found at the screen point.
+                        let graphics = identifyResult.graphics
+                        if graphics.isNotEmpty {
+                            let postfix = graphics.count > 1 ? "s" : ""
```

Comment:
Is this needed? Can the count ever be more than one?

Reply:
In this use case, no, because there is only one graphic on the map. If the user tapped where multiple graphics were overlapping, then yes. I think it would be good to leave it to show that use case and because design/other samples do it that way. (This sample would have been better with multiple graphics, but I am trying to follow the design as closely as possible.)

Reply:
> I think it would be good to leave it to show that use case

It is good to want to show users how to do things, but we also want to show users how to do things the right way. While the samples app isn't localized, the user's app might be, so we want to make sure that we are modeling best practices when it comes to localization. When localizing strings, concatenating them is a big no-no. While "graphic" + "s" = "graphics" may work in English, that won't work in many other languages. If users were to copy and paste this code into their own localized app (turning both strings into localized strings), it would introduce a bug. Try to keep that in mind when displaying anything localizable to the user.